packages/react-native-renderer/src/ReactNativeFiberHostComponent.js

```diff
@@ -33,12 +33,18 @@ import {
 class ReactNativeFiberHostComponent {
   _children: Array<Instance | number>;
   _nativeTag: number;
+  _internalFiberInstanceHandle: Object;
   viewConfig: ReactNativeBaseComponentViewConfig<>;
 
-  constructor(tag: number, viewConfig: ReactNativeBaseComponentViewConfig<>) {
+  constructor(
+    tag: number,
+    viewConfig: ReactNativeBaseComponentViewConfig<>,
+    internalInstanceHandle: Object,
+  ) {
     this._nativeTag = tag;
     this._children = [];
     this.viewConfig = viewConfig;
+    this._internalFiberInstanceHandle = internalInstanceHandle;
```

Comment:
Is this only used in DEV? If yes, please call it `this._internalFiberInDEV` or something and make it DEV-only to avoid using an extra field.

Reply:
We have the same field in prod (used in prod) on the fabric host component. Do you feel like making it dev is the right choice knowing it’s on the fabric host component too? It seems reasonable to make it dev only since it is only needed in dev afaik but just checking.

Reply:
If we don't use some field in PROD (whether or not it exists on other places), it's generally a good practice to make it DEV only to reduce the memory usage. If only due to the extra pointer.